sdl2promela/sdl/type_info.py

```diff
@@ -0,0 +1,40 @@
+from typing import Dict
+
+from sdl2promela.utils import Asn1Type
+
+
+class TypeInfo:
+
+    all_types: Dict[str, str]
+
+    def __init__(self):
+        self.all_types = {}
+
+    def fix_type_name(self, name: str) -> str:
+        if name in self.all_types:
+            return self.all_types[name]
+        return name
+
+
+# def create_type(name: str, has_parent: bool, t: Any)
+
+
+def build_type_info(all_types: Dict[str, Asn1Type]) -> TypeInfo:
+    type_info = TypeInfo()
+
+    for name, t in all_types.items():
```

Comment:
Use type_name instead of name? t does not mean much.

Reply:
Sorry, asn1_type?

Reply:
Changed